storagenode/peer.go

```diff
@@ -258,6 +258,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config Config, ver
 			peer.Transport,
 			peer.Kademlia.Service,
```

Comment:
Is kademlia still used here somewhere, maybe it can be removed?

Reply:
it is not used anymore. I removed it